src/components/DateRangePickerInputController.jsx

```diff
@@ -232,6 +242,11 @@ export default class DateRangePickerInputController extends React.PureComponent
 
   onStartDateFocus() {
     const { disabled, onFocusChange } = this.props;
+
+    this.setState({
```

Comment:
I'm not convinced we need this state value. we can use the `isStartDateFocused` and `isEndDateFocused` props instead

Reply:
Neither was I, I was going to investigate it a bit more after I figured out the tabbing issues (which I think I'll need to do some somewhat-hacky stuff to get working, unfortunately). Thanks :)